src/reducers/index.js

```diff
@@ -0,0 +1,24 @@
+
+import { ADD_TODO } from '../actions';
+
+const initialState = {
+  toDoList: []
+};
+
+export default function toDoApp(state = initialState, action) {
```

Comment:
You can rename this to `todoReducer` because it is a reducer.

Reply:
Also, you can keep it in a separate file (`todo.reducer.js` or something like that)